gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -181,6 +202,22 @@ def get_predetermined_qc(version: str = CURRENT_VERSION, test: bool = False):
         return v4_predetermined_qc.versions[version]
 
 
+def get_joint_qc_mt(version: str = CURRENT_VERSION, test: bool = False):
```

Comment:
See https://the-tgg.slack.com/archives/C018PB68VU5/p1664206854601279 for my thoughts on how we should start handling test resources instead of making functions for each of them.